components/NavigationBar.vue

```diff
@@ -158,31 +175,42 @@ export default {
   transition: 120ms linear background-color;
   background-color: transparent;
 
-  .k-navigation-bar.x-show-background & {
+  .navigation-bar.x-show-background & {
     background-color: var(--colors-background);
   }
 
-  .k-navigation-bar.x-scrolled & {
-    .k-navigation-bar__title {
+  .navigation-bar.x-scrolled & {
+    .navigation-bar__title {
       opacity: 1;
       transform: translateY(0);
     }
   }
 }
 
-.k-navigation-bar__toggle {
+.navigation-bar__toggle {
   display: none;
 }
 
-.k-navigation-bar__logo {
+.navigation-bar__logos {
+  margin-right: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .navigation-bar__language {
+    display: none;
+  }
+}
+
+.navigation-bar__logo {
   height: 70px;
   min-height: 70px;
   width: 46.3833px;
   min-width: 46.3833px;
   margin-right: 20px;
 }
 
-.k-navigation-bar__title {
+.navigation-bar__title {
   font-size: 1.5rem;
```

Comment:
font size should be smaller because if you visit the German version of contribute the content of the navigation bar is bigger than its size